src/components/modals/FioCreateHandleModal.tsx

```diff
@@ -38,6 +43,22 @@ export const FioCreateHandleModal = (props: Props) => {
     bridge.resolve(true)
   })
 
+  React.useEffect(() => {
```

Comment:
I don't think this % stuff is necessary and it's more regression risk too late in the release.